ceph-releases/jewel/daemon/check_zombie_mons.py

```diff
@@ -23,25 +30,30 @@ def extract_mons_from_monmap():
             mons[m.group(2)] = m.group(1)
     return mons
 
+
 def extract_mons_from_kubeapi():
     kubemap = subprocess.check_output(kubectl_command, shell=True)
     return json.loads(kubemap)
 
+
 current_mons = extract_mons_from_monmap()
 expected_mons = extract_mons_from_kubeapi()
 
-print "current mons:", current_mons
-print "expected mons:", expected_mons
+print 'current mons:{}'.format(current_mons)  # noqa E999
```

Comment:
Ignoring E999 is probably not a good idea. If the line is changed later and there actually is an error, it might not be caught. From my reading, it appears that E999 is usually due to environment issues rather than code issues. Make sure you are using flake8 with python3 and not python2.